shared-bindings/displayio/Bitmap.c

```diff
@@ -62,14 +62,11 @@
 //|         ...
 STATIC mp_obj_t displayio_bitmap_make_new(const mp_obj_type_t *type, size_t n_args, size_t n_kw, const mp_obj_t *all_args) {
     mp_arg_check_num(n_args, n_kw, 3, 3, false);
-    uint32_t width = mp_obj_get_int(all_args[0]);
-    uint32_t height = mp_obj_get_int(all_args[1]);
-    uint32_t value_count = mp_obj_get_int(all_args[2]);
+    uint32_t width = mp_arg_validate_int_min(mp_obj_get_int(all_args[0]), 1, MP_QSTR_width);
+    uint32_t height = mp_arg_validate_int_min(mp_obj_get_int(all_args[1]), 1, MP_QSTR_height);
+    uint32_t value_count = mp_arg_validate_int_min(mp_obj_get_int(all_args[2]), 1, MP_QSTR_value_count);
```

Comment:
These should be validated for maximum as well.  Surely 32767 is a reasonable maximum (as we have 16-bit types anyhow), and it ensures that `width * height * bytes_per_pixel` cannot overflow a size_t, so there are no potential overflows when computing how much memory to allocate.

Reply:
This is handled now in the latest commits using range and a max of 32767. Thank you!